### ISSUE TITLE
Opening a dropdown panel for list properties throws an error

### ISSUE THREAD

**Issue by psmyrek** [ckeditor/ckeditor5#12337]
## 📝 Provide detailed reproduction steps (if any)

This error occurs only in **lists without styles**, but configured with extra properties: start index, or reversed order, or both.

1. Open "List start index" demo: https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/features/lists/lists.html#demo-3
2. Open the list dropdown panel to change the start index.

### ✔️ Expected result

No error in the console.

### ❌ Actual result

```
Uncaught TypeError: Cannot read properties of null (reading 'children')
    at eval (listpropertiesui.js:207:129)
    at dropdownView.on.priority (utils.js:289:24)
    at DropdownView.fire (emittermixin.ts:168:47)
    at DropdownView.set [as isOpen] (observablemixin.ts:87:30)
    at SplitButtonView.eval (dropdownview.js:254:16)
    at SplitButtonView.fire (emittermixin.ts:168:47)
    at fireDelegatedEvents (emittermixin.ts:443:17)
    at ButtonView.fire (emittermixin.ts:187:25)
    at TemplateToBinding.eval (buttonview.js:162:12)
    at ProxyEmitter.callback (template.js:973:11)
```

https://user-images.githubusercontent.com/56868128/186606635-6442aad8-84df-4ae9-9d21-b58ac8f012c3.mp4

### 💡 Remarks

- The first version where the error occurs is 35.0.1. This error does not exist in version 34.2.0 and earlier.
- The editor is still fully functional after the error is thrown. The "list start index" and "reversed list" features work fine.
- The same error occurs in the "Reversed list" demo: https://ckeditor5.github.io/docs/nightly/ckeditor5/latest/features/lists/lists.html#demo-4.
- The same error occurs in the manual test for the list properties: http://localhost:8125/ckeditor5-list/tests/manual/list-properties.html (scroll down to the "Without styles, just extra properties" section).

## 📃 Other details

* Browser: Firefox, Chrome
* OS: Win 11
* First affected CKEditor version: 35.0.1
* Installed CKEditor plugins: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by oleq** [comment]
It's probably related to the logic responsible for focusing dropdown children when the dropdown gets open. I'd there's a `focus()` method missing on some child in this case.

**Comment by mmotyczynska** [comment]
This is because we try to focus the first child of `stylesView`, but the `stylesView` does not exist in this example.

[https://github.com/ckeditor/ckeditor5/blob/7dfb504a3339fe287627bf422cae8c2a295303bf/packages/ckeditor5-list/src/listproperties/listpropertiesui.js#L188](https://github.com/ckeditor/ckeditor5/blob/7dfb504a3339fe287627bf422cae8c2a295303bf/packages/ckeditor5-list/src/listproperties/listpropertiesui.js#L188)

However there is already some other logic that focuses the first focusable view in a dropdown and when we remove the above line of code then the feature starts to work as expected.

The only thing that seems not to work properly for me is that when `startIndexFieldView` and `reversedSwitchButtonView` are disabled, they are still focused by default (of course when the `stylesView` is missing).

**Comment by oleq** [comment]
> However there is already some other logic that focuses the first focusable view in a dropdown and when we remove the above line of code then the feature starts to work as expected.

Could you prototype this as a draft PR? I'd be glad to see it in action.

> The only thing that seems not to work properly for me is that when `startIndexFieldView` and `reversedSwitchButtonView` are disabled, they are still focused by default (of course when the `stylesView` is missing).

This isn't bad. A disabled UI control can still be focused, if only to make the UI predictable.

**Comment by mmotyczynska** [comment]
> Could you prototype this as a draft PR? I'd be glad to see it in action.

Done.

**Comment by oleq** [comment]
[I left some comments under the PR](https://github.com/ckeditor/ckeditor5/pull/12350#pullrequestreview-1085566818).

**Comment by oleq** [comment]
Closed in ckeditor/ckeditor5#12350.
